relay_messages.go

```diff
@@ -91,13 +91,22 @@ func (cr lazyCallRes) OK() bool {
 	return cr.Payload[_resCodeIndex] == _resCodeOK
 }
 
+type keyVal struct {
+	key []byte
+	val []byte
+}
+
 type lazyCallReq struct {
 	*Frame
 
 	caller, method, delegate, key, as []byte
 
+	headerStartOffset              int
+	headerEndOffset                int
+	checksumType                   ChecksumType
```

Comment:
nit: play with the size of this struct and check whether this is the right place to store this value. I'd guess that putting this next to `isArg2Fragmented` will make the struct smaller (ints are aligned to 8-byte boundaries in 64-bit, so putting a 1 byte "wastes" 7 bytes after it, vs packing the small 1 byte values together)

Reply:
good call! placed `IsArg2Fragmented` and `ChecksumType` together reduced the struct by 8 bytes.